src/main/java/com/android/volley/toolbox/HurlStack.java

```diff
@@ -77,7 +77,7 @@ public HurlStack(UrlRewriter urlRewriter) {
     }
 
     /**
-     * @param urlRewriter Rewriter to use for request URLs
+     * @param urlRewriter      Rewriter to use for request URLs
```

Comment:
I'd prefer if you revert the formatting changes for now. This format doesn't match what I think we'd be likely to use in the future (google-java-format), and I'd rather handle that in a more coordinated matter.